utils/local-fbsd-api.sh

```
@@ -0,0 +1,23 @@
+cd cla-backend-go
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)